### PR TITLE
Added note about arrays

### PR DESCRIPTION
**Pull request by franciscop**
Added note about arrays being returned as JSON.

### CODE REVIEW

**Comment by fl0w** [comment]
This occurs because of the resent header parser change in core if I'm not mistaken, correct?

**Comment by franciscop** [comment]
@fl0w I am sorry I got some trouble understanding the English in that sentence, could you rephrase it please? What header parser change in core? (is that something that happened in Koa? or something I don't understand?)

If it makes it any more clear, I wrote an issue here that explains why it happens: https://github.com/koajs/koa/issues/965
